packs/feats/widen-spell.json

```diff
@@ -104,6 +104,79 @@
                 "priority": 119,
                 "property": "area-size",
                 "value": 10
+            },
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "spellshape:widen-spell",
+                    "item:area:type:burst",
+                    {
+                        "gte": [
+                            "item:area:size",
+                            10
+                        ]
+                    }
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Spellshape.WidenSpell.Burst"
+                    }
+                ]
+            },
```

Comment:
Hm I personally don't think we need an addendum if the thing Just Works™, like Widen Spell does.

Reply:
So drop the description for everything?

Reply:
Yeah, I think the text being there could make it seem like you need to adjust the size yourself, when in reality it just works

Reply:
I like that the pop-up description reminds me that the ability/item has been used on top of the spell.

Would love to have item alterations on action cost to account for this.

Reply:
Once that happens, we'd remove those description alterations and have vestigial stuff in the localization file that can't just be deleted without migrating. I don't think they bring enough value to deal with that, specially when the automation works completely.